outport/process/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -125,6 +131,28 @@ func (aap *alteredAccountsProvider) processMarkedAccountData(
 	return nil
 }
 
+func (aap *alteredAccountsProvider) loadUserAccount(addressBytes []byte, options shared.AlteredAccountsOptions) (state.UserAccountHandler, error) {
+	var account vmcommon.AccountHandler
+	var err error
+
+	if !options.WithCustomAccountsRepository {
```

Comment:
I forgot, why do we need to support both approaches?

Reply:
because normally altered accounts are used to index all touched accounts during a block. Do you think we can use accountsrepository for both needs?